frontend/tablet/src/screens/RoomDetailScreen/Header.tsx

```diff
@@ -1,14 +1,25 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { View } from 'react-native'
+import moment from 'moment'
 
 import { Typography, Button, Grid } from '../../components'
 
 const Header: React.FC = () => {
+  const [time, setTime] = useState('')
+  const [date, setDate] = useState('')
+
+  useEffect(() => {
+    setInterval(() => {
+      setTime(moment().add(1, 'hours').format('hh:mm'))
+      setDate(moment().format('dddd, MMM DD'))
+    }, 1000)
+  })
```

Comment:
Why do you need 2 states for this? It's enough to have `const [dateTime, setDateTime] = useState(new Date())` and then update with `new Date()` every 1 second. Only use formatting in the render, keep pure data in the state.